rust_src/src/buffers.rs

```diff
@@ -105,6 +110,90 @@ pub type LispBufferRef = ExternalPtr<Lisp_Buffer>;
 pub type LispOverlayRef = ExternalPtr<Lisp_Overlay>;
 
 impl LispBufferRef {
+    pub fn create_new(name: LispStringRef) -> LispBufferRef {
+        if name.is_empty() {
+            error!("Empty string for buffer name is not allowed");
+        }
+
+        let mut b = unsafe { ExternalPtr::new(allocate_buffer()) };
```

Comment:
why not `LispBufferRef::new` ? here

Reply:
Because it would conflict with `ExternalPtr`'s `new`

Reply:
Though following the standard of a lot of other functions that initialize objects, it would probably be more in line with convention to name it `make_new` or just `make`